### PR TITLE
Example actions

### PR DESCRIPTION
**Pull request by ForTehLose**
This PR includes:

- helpers for actions in the XR Utils crate
- an example for openxr using the xr utils action helpers
- basic flycam in the action example

### CODE REVIEW

File: crates/bevy_xr_utils/Cargo.toml [hunk 1]
```diff
@@ -8,3 +8,5 @@ edition = "2021"
 [dependencies]
 bevy.workspace = true
 bevy_xr.path = "../bevy_xr"
+openxr = "0.18.0"
+bevy_openxr.path = "../bevy_openxr"
```
Comment (TheButlah) [inline crates/bevy_xr_utils/Cargo.toml:12]: Files should end in newline

File: crates/bevy_xr/src/actions.rs [hunk 2]
```diff
@@ -21,6 +21,7 @@ impl<A: Action> Plugin for ActionPlugin<A> {
     }
 }
 
+#[derive(Debug)]
```
Comment (TheButlah) [inline crates/bevy_xr/src/actions.rs:24]: Any reason not to implement eq, partial eq, hash, clone, and copy? I would expect almost every public type to have these derived. Except copy, and maybe clone.

File: crates/bevy_xr_utils/src/xr_utils_actions.rs [hunk 3]
```diff
@@ -0,0 +1,360 @@
+use bevy::prelude::*;
```
Comment (TheButlah) [inline crates/bevy_xr_utils/src/xr_utils_actions.rs:1]: Add a module-level doc comment explaining the role of this module.

**Review by TheButlah — COMMENTED** [review]
Don't have context to do a proper review rn, but here's some nits :3

File: crates/bevy_xr/src/actions.rs (hunk 2 above)
Comment (ForTehLose) [reply crates/bevy_xr/src/actions.rs:24]: I just wanted debug so I could easily print it.

File: crates/bevy_openxr/examples/actions.rs [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+    });
+    // cube
+    commands.spawn(PbrBundle {
+        mesh: meshes.add(Cuboid::new(1.0, 1.0, 1.0)),
+        material: materials.add(Color::rgb_u8(124, 144, 255)),
+        transform: Transform::from_xyz(0.0, 0.5, 0.0),
+        ..default()
+    });
+    // light
+    commands.spawn(PointLightBundle {
```
Comment (MalekiRe) [inline crates/bevy_openxr/examples/actions.rs:53]: No point lights, add an ambient light instead as a resource.

**Review by MalekiRe — CHANGES_REQUESTED** [review]
LGTM other than the point light in example

File: crates/bevy_openxr/examples/actions.rs (hunk 4 above)
Comment (ForTehLose) [reply crates/bevy_openxr/examples/actions.rs]: I have changed to an ambient light for better performance like you asked.

**Review by MalekiRe — APPROVED** [review]
LGTM

**Pull request merged**